### PR TITLE
BGE-LC bug hangs on delete row

### PR DESCRIPTION
**Pull request by ptewson-sfdo**
Change the const declaration to var to resolve the bug (fix discovered and suggested by @randi274)

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by ptewson-sfdo** [comment]
**lurch: attach W-030360

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007fQGtQAM" target="_blank">W-030360</a>

File: src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryController.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
                 action.setCallback(this, function (response) {
                     const state = response.getState();
                     if (state === 'SUCCESS') {
-                        const response = JSON.parse(response.getReturnValue());
+                        var response = JSON.parse(response.getReturnValue());
```
Comment (randi274) [inline src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryController.js]: Any opinion on creating a new variable instead of recreating response?

Comment (ptewson-sfdo) [reply src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryController.js]: Yes, STRONG opinion that this is the right way to do it :)

**Comment by cpose** [comment]
QA Approved

**Pull request merged**